### DISCUSSION TITLE
1.1.36

### DISCUSSION THREAD

**Discussion by elihwyma** [wstyres/Zebra#2059] [Releases]
- Fix installing packages from local deb file

<hr /><em>This discussion was created from the release <a href='https://github.com/zbrateam/Zebra/releases/tag/v1.1.36'>1.1.36</a>.</em>

**Comment by liudayu** [comment]
It deleted all my repo lists

**Reply by iAdam1n** [reply]
Do you have any tweaks installed which interfere with Zebra at all? I just updated it on 4 of my devices and not one lost repo list.

**Reply by elihwyma** [reply]
Can you provide anymore info on the issue

**Reply by liudayu** [reply]
Opened zebra app -> found the zebra update -> tried to install 3 new tweaks first -> found them not installed (button just says done instead of respring) -> installed 3 tweak successfully one by one without pressing the respring button, pressed done every time) -> install zebra update -> close zebra and reopen, repo lists gone.

**Reply by Nattawat1701** [reply]
Opened zebra app -> found the zebra update -> tried to install 3 new tweaks first -> found them not installed (button just says done instead of respring) -> installed 3 tweak successfully one by one without pressing the respring button, pressed done every time) -> install zebra update -> close zebra and reopen, repo lists gone.
